docs/platforms/apple/common/tracing/instrumentation/automatic-instrumentation.mdx

```diff
@@ -333,13 +333,13 @@ _(New in version 8.4.0)_
 
 By adding a span for a view controller when it's loaded, time to initial display (TTID) provides insight into how long it takes for your view controller to launch and draw its first UI frame. The SDK sets the span operation to `ui.load.initial-display` and the span description to the view controllers's name, followed by `initial display` - for example, `MainViewController initial display`.
 
-The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment.
+The span starts when the view of a view controller is loaded, and there is no other view controller transaction happening at the moment. The span finishes after the view appeared on the screen. The following chart shows how time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:
 
-The span finishes after the view appeared on the screen.
+![](./img/ttid-ttfd-concept.png)
 
-The following chart shows how time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) correlate to transitions between activities:
+Since Cocoa SDK version 8.33.0, the SDK doesn't create time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) for UIViewControllers presented in the background because the logic requires UI frames to be drawn.
 
-![](./img/ttid-ttfd-concept.png)
+The SDK doesn't create time to initial display (TTID) and [time to full display (TTFD)](#time-to-full-display) spans for UIViewControllers presented after receiving the [UIApplicationDidEnterBackgroundNotification](https://developer.apple.com/documentation/uikit/uiapplicationdidenterbackgroundnotification), because the logic requires UI frames to be drawn.
```

Comment:
IIUC this was always the case, so this is not something we changed in 8.33.0, correct? if so, I'd put this (more generic) information above the note about the change in 8.33.0

Reply:
No, we changed this in 8.33.0. In previous versions, the SDK created long-lasting spans for TTID.

Reply:
Ah ups, forgot to mention 8.33.0 here. Now I get it.